apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/varselbrev/BrevDataMapperRedigerbartUtfallVarsel.kt

```diff
@@ -24,7 +30,23 @@ object BrevDataMapperRedigerbartUtfallVarsel {
 
         SakType.OMSTILLINGSSTOENAD ->
             if (revurderingsaarsak == Revurderingaarsak.AKTIVITETSPLIKT) {
-                OmstillingsstoenadAktivitetspliktVarselUtfall()
+                val erTidligereFamiliepleier = detaljertBehandling?.tidligereFamiliepleier?.svar == true
+
+                val doedsdatoEllerOpphoertPleieforhold =
+                    if (erTidligereFamiliepleier) {
+                        detaljertBehandling?.tidligereFamiliepleier!!.opphoertPleieforhold!!
+                    } else {
+                        grunnlag
+                            ?.hentAvdoede()
+                            ?.singleOrNull()
+                            ?.hentDoedsdato()
+                            ?.verdi
+                    }
+
+                val virk = detaljertBehandling!!.virkningstidspunkt?.dato
+                val er6mndvarsel = virk!!.isAfter(YearMonth.from(doedsdatoEllerOpphoertPleieforhold!!.plusMonths(12)))
```

Comment:
Blir dette for komplisert?

Reply:
Blir det riktig da, nå sier du hvis `virk` er _etter_ `dødsdato/pleiedato` pluss 12 mnd så er det 6 måneders varsel som skal brukes. Er det ikke `12 mnd` da?